src/pages/docs/concepts-invocations.mdx

```diff
@@ -0,0 +1,223 @@
+through _invocations_.
+
+An _invocation_ is the act of invoking an exported function on a specific 
+worker, by supplying the function with its required input parameters. If the 
+invoked function has a return value, the invocation will return the value.
+
+Perhaps surprisingly for those new to Golem, every invocation occurs on a 
+_specific worker_, rather than more generally on a component. If an attempt is 
+made to invoke a function on a worker that does not exist yet, it will be 
+created through the act of invocation.
+
+Invocation is such a fundamental aspect of building useful applications on 
+Golem that in this section, we will discuss the origin of invocations, and 
+provide high-level instructions on how to perform invocations during 
+development, testing, and production scenarios.
+
+# Structure of Invocations
+
+The structure of invocations on workers derives from the WASM component model, 
+which defines the constructs that make up the public APIs of components.
+
+The public API of a component is defined using WASM Interface Types (WIT), a 
+text format analogous to protocol buffers (protobuf).
+
+Through WIT, components may export functions, which can be called from other 
+components (as per the component model), but which Golem also makes available 
+for remote invocation through multiple protocols.
+
+Understanding the structure of invocations, then, comes down to understanding 
+the structure of component model types, as defined by WIT.
+
+In the next section, you will learn more about this structure.
+
+## Basic Invocations 
+
+Assume we have created a component designed to provide durable counters, whose 
+public API is defined by the following WIT:
+
+```
+package demo:counter;
+
+interface counters {
+  get-total-counters: func() -> u32;
+
+  increment-counter: func(index: s32) -> s32;
+
+  get-counter: func(index: s32) -> s32;
+}
+```
+
+This API defines three standalone functions, named `get-total-counters`, 
+`increment-counter`, and `get-counter`.
+
+In order to perform an invocation on a worker created from this component, we 
+will need two specify two things:
+
+1. The fully qualified name of the function that we are invoking on the worker.
+2. The parameters to invoke the function with, which must have the correct 
+   structure.
+
+The structure of the name of the function is fully standardized across all 
+supported invocation protocols, and is as follows:
+
+```
+<package-name>/<interface-name>.{<function-name>}
+```
+
+So, in the preceding example, if we wish to invoke the `get-total-counters` 
+function on some worker, we must utilize the following fully qualified name:
+
+```
+demo:counter/counters.{get-total-counters}
+```
+
+This name identifies the `get-total-counters` function, unique from any other 
+function that is exported by the component.
+
+You will learn how to specify parameters later, when the different supported 
+protocols are introduced.
+
+## Resource Invocations
```

Comment:
Is it intentional that you don't mention the explicit resource usage syntax? Calling constructors, methods, and drop explicitly. Not sure when would I choose to use those instead of the other one, but I think it makes sense to introduce it for example because when you list the component export's with `golem-cli` that's how you see them.

Reply:
It is not intentional. Where can I find examples?

Reply:
The `function-names.mdx` contains it